tasks/omnibus.py

```diff
@@ -314,6 +313,7 @@ def build(
             if use_remote_cache and ctx.run(f"git -C {omnibus_cache_dir} tag -l").stdout != cache_state:
                 ctx.run(f"git -C {omnibus_cache_dir} bundle create {bundle_path} --tags")
                 ctx.run(f"{aws_cmd} s3 cp --only-show-errors {bundle_path} {git_cache_url}")
+                bundle_dir.close()
```

Comment:
I think this would need to be:

```suggestion
                bundle_dir.cleanup()
```

Also, I think the cleanup would happen when the object is garbage-collected ([ref](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory.cleanup))

Reply:
Ah I missed your comment but ended up doing the change anyway, since as you noticed, there's no such function on a `TemporaryDirectory` object.
I'm open to let garbage collection handle this, but I'm a bit biased against having resources cleanup ran at an unknown time